### PR TITLE
updates types for getAllAWSRoleTypes

### PR DESCRIPTION
**Pull request by americk0**

### CODE REVIEW

**Review by webbbarker — DISMISSED** [review]
👍

File: src/alks.js [hunk 1]
```diff
@@ -223,7 +223,7 @@ class alks {
     * })
     */
   getAllAWSRoleTypes(props) {
-    const url = props.getDynamicValues ? 'allAwsRoleTypes' : 'allAwsRoleTypes?getDynamicValues=true'
```
Comment (webbbarker) [inline src/alks.js]: I'll take "Why ternary operators can obscure logic for 1000, Alex".  😄

**Review by webbbarker — APPROVED** [review]
LGTM 👍

File: src/alks.js (hunk 1 above)
Comment (americk0) [reply src/alks.js]: To be fair my dumb ass probably would've written 
```javascript
let url;
if (props.getDynamicValues) {
  url = 'allAwsRoleTypes'
} else {
  url = 'allAwsRoleTypes?getDynamicValues=true'
}
```
and gotten it backwards even if I didn't use the ternary 🤦

**Pull request merged**